server/models/meshmodel/core/register.go

```diff
@@ -211,7 +211,7 @@ func getCRDsFromManifest(manifest string, arrAPIResources []string) []crdRespons
 				continue
 			}
 			if kind == resource {
-				crd, err := fieldVal.MarshalJSON()
+				crd, err := fieldVal.LookupPath(cue.ParsePath("properties.spec")).MarshalJSON()
```

Comment:
Check warning on line 80 in server/models/meshmodel/core/register.go

Lint Issue
/ golangci-lint (1.19, ubuntu-22.04)

unused-parameter: parameter 'ctx' seems to be unused, consider removing or renaming it as _ (revive)